app/providers.tsx

```diff
@@ -1,13 +1,8 @@
 "use client";
 
-import { CacheProvider } from "@chakra-ui/next-js";
 import { ChakraProvider } from "@chakra-ui/react";
 import customTheme from "../styles/theme";
 
 export function Providers({ children }: { children: React.ReactNode }) {
-  return (
-    <CacheProvider>
-      <ChakraProvider theme={customTheme}>{children}</ChakraProvider>
-    </CacheProvider>
-  );
+  return <ChakraProvider theme={customTheme}>{children}</ChakraProvider>;
```

Comment:
looking into whether or not this omission is causing Flash Of Unstyled Content (FOUC) on preview build (and thus would also affect prod)